tests/testthat/test-uploadPackage.R

```diff
@@ -0,0 +1,84 @@
+
+test_that("Uploaded packages show up in summary table", {
+  # delete app DB if exists to ensure clean test
+  if (file.exists(test_path("test-apps", "database.sqlite"))) {
+    file.remove(test_path("test-apps", "database.sqlite"))
+  }
+
+  # set up new app driver object
+  app <- AppDriver$new(app_dir = test_path("test-apps"))
+
+  # test package data to upload
+  test_csv <- system.file("extdata", "upload_format.csv", package = "riskassessment")
+
+  # upload file to application
+  app$upload_file(
+    `upload_package-uploaded_file` = test_csv
+  )
+
+  # wait for table to be shown
+  app$wait_for_value(
+    output = "upload_package-upload_pkgs_table",
+    ignore = list(NULL),
+    timeout = 30 * 1000 # CI keeps failing here...
```

Comment:
One thing we can do is to not upload 3 packages. Shouldn't one be sufficient to test this functionality?

Reply:
Dropped down to one package in [b7f4d32](https://github.com/pharmaR/risk_assessment/pull/322/commits/b7f4d32276b90322ad9e867273e755e708f8dfd3)